net/netifyd/Makefile

```diff
@@ -0,0 +1,83 @@
+# Netify Agent
+# Copyright (C) 2016-2018 eGloo, Incorporated
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netifyd
+PKG_VERSION:=2.67
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Darryl Sokoloski <darryl@egloo.ca>
+PKG_LICENSE:=GPL-3.0+
+
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+PKG_CONFIG_DEPENDS:=CONFIG_LIBCURL_ZLIB
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.com/netify.ai/public/netify-agent.git
+PKG_SOURCE_DATE:=2018-07-04
+PKG_SOURCE_VERSION:=d05494e074495909837186bd7fad11ba89022d92
+PKG_MIRROR_HASH:=9ed3232739d162985a4f6f6e156792a0bed572f1a52a4618cb96604c0b0c21bf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/netifyd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Netify Agent
+  URL:=http://www.netify.ai/
+  DEPENDS:=+libcurl +libmnl +libnetfilter-conntrack +libjson-c +libpcap +zlib +libpthread +libstdcpp
+endef
+
+define Package/netifyd/description
+The [Netify](https://www.netify.ai/) Agent is a deep-packet inspection server.  The Agent is built on top of [nDPI](http://www.ntop.org/products/deep-packet-inspection/ndpi/) (formerly OpenDPI) to detect network protocols and applications.  These detections can be saved locally, served over a UNIX or TCP socket, and/or "pushed" (via HTTP POSTs) to a remote third-party server.  Flow metadata, network statistics, and detection classifications are stored using JSON encoding.
+
+Optionally, the Netify Agent can be coupled with a [Netify Cloud](https://www.netify.ai/) subscription for further cloud processing, historical storage, machine-learning analysis, event notifications, device detection/identification, along with the option (on supported platforms) to take an active role in policing/bandwidth-shaping specific network protocols and applications.
```

Comment:
Lines too long.  Personally I prefer this section to be more terse and plain text only as compared to the markdown syntax.

Please add it here or in another README.md file about how people are expected to use this package.  This is very important.